extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -691,8 +691,9 @@ export async function newDeployment(
         step: 0,
         totalSteps: 0,
         value: currentName,
-        prompt: "Enter a Unique Nickname for your Credential.",
-        placeholder: "example: Posit Connect",
+        prompt:
+          "Enter a unique nickname for this server and API key combination.",
+        placeholder: "Posit Connect",
```

Comment:
My comment above applies here as well: `Enter a unique nickname for this server`